sources/packages/backend/libs/utilities/src/parallel-processing-utils.ts

```diff
@@ -1,4 +1,19 @@
-import * as os from "os";
+/**
+ * Number of parallel processes allowed to be started at same time.
+ */
+export enum ParallelIntensity {
+  /**
+   * Used for regular processes that can be executed in parallel where the
+   * waiting time is expected to be low, for instance, for regular quick
+   * DB access.
+   */
+  Regular = 2,
```

Comment:
It should be the other way round right for parallel processing?

I mean for short lived processes more accepted parallel threads and long processed with less parallel threads.

Reply:
Longer processes (I/O processes) will leave the processor free to take more "processing" work, right?

Reply:
Comments adjusted 😉 
Thanks for the nice conversation.